arch/arm/src/dm320/dm320_decodeirq.c

```diff
@@ -45,7 +45,8 @@ uint32_t *arm_decodeirq(uint32_t *regs)
   struct tcb_s *tcb = this_task();
 
 #ifdef CONFIG_SUPPRESS_INTERRUPTS
-  up_set_current_regs(regs);
+  tcb->xcp.regs = regs;
+  arm_set_irq_flag(true);
```

Comment:
I prefer the previous naming or up_set_interrupt_context() to arm_set_irq_flag()

Reply:
done